graphbolt/include/graphbolt/cuda_ops.h

```diff
@@ -162,16 +164,21 @@ torch::Tensor ExclusiveCumSum(torch::Tensor input);
 torch::Tensor UVAIndexSelectImpl(torch::Tensor input, torch::Tensor index);
 
 /**
- * @brief CSRToCOO implements conversion from a given indptr offset tensor to a
- * COO format tensor including ids in [0, indptr.size(0) - 1).
+ * @brief CSCToCOO implements conversion from a given indptr offset tensor to a
+ * COO format tensor. If original_column_node_ids is not given, it is assumed to
+ * be equal to torch::arange(indptr.size(0) - 1, dtype=output_dtype).
  *
- * @param input          A tensor containing IDs.
- * @param output_dtype   Dtype of output.
+ * @param indptr                 The indptr offset tensor.
+ * @param output_dtype           Dtype of output.
+ * @param num_edges              Optional number of edges, equal to indptr[-1].
+ * @param original_column_node_ids  Optional original row ids for indptr.
  *
- * @return
- * - The resulting tensor with output_dtype.
+ * @return The resulting tensor with output_dtype.
  */
-torch::Tensor CSRToCOO(torch::Tensor indptr, torch::ScalarType output_dtype);
+torch::Tensor ExpandIndptrImpl(
```

Comment:
Please align the augments between the C++ api and python api, including order and name.

Reply:
I need to keep dtype as the second argument in the C++. Do you think I can change the python API instead to have dtype as the 2nd argument?

Reply:
Let me just go ahead and make C++ conform to python actually.